.github/workflows/ruby.yml

```diff
@@ -1,35 +1,22 @@
-# This workflow uses actions that are not certified by GitHub.
-# They are provided by a third-party and are governed by
-# separate terms of service, privacy policy, and support
-# documentation.
-# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
-# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
-
 name: Ruby
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
```

Comment:
I'm not too familiar with github workflows. Does this mean, the pipeline can be run on all branches except main?

If yes, would we want to run the pipeline on main too, after merging a PR for example?

Reply:
I'm not too sure but I think it means that it runs on main only when merge into main or pushed up to main.